src/Build/BackEnd/Components/SdkResolution/SdkResolverManifest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using System.Runtime.Serialization;
+using System.Xml;
+
+namespace Microsoft.Build.BackEnd.SdkResolution
+{
+    /// <summary>
+    /// Serialization contract for an SDK Resolver manifest
+    /// </summary>
+    [DataContract(Name = "SdkResolver", Namespace = "")]
```

Comment:
Why this kind of XML serialization rather than just reading the XML manually, as we do elsewhere?

Reply:
This is used for the AssemblyFolder config stuff. No real reason other than simplicity. I'm not opposed to just reading the xml manually.